2016-2017_Starstruck/Module Prototypes/include/LCD.h

```diff
@@ -19,7 +21,7 @@ typedef struct MenuItem {
  * This function should be called from initialize()
  * DO NOT initialize the LCD separately
  */
-MenuItem *lcdmInit();
+MenuItem *lcdmInit(FILE *lcdPort);

```

Comment:
this is everywhere why is there an asterisk here? like are you passing the pointer to the MenuItem around instead of the MenuItem itself?